.stylelintrc.json

```diff
@@ -0,0 +1,32 @@
+{
+    "extends": "stylelint-config-standard",
+    "rules": {
+      "color-named": "never",
+      "color-function-notation": "modern",
+      "declaration-block-no-redundant-longhand-properties": true,
+      "font-family-name-quotes": "always-where-recommended",
+      "font-family-no-missing-generic-family-keyword": [
+        true,
+        {
+          "ignoreFontFamilies": ["FontAwesome", "kbase-icons"]
+        }
+      ],
+      "length-zero-no-unit": true,
+      "no-descending-specificity": null,
+      "selector-max-compound-selectors": 5,
+      "selector-max-class": 5,
```

Comment:
Arbitrarily chosen -- we do have existing selectors that break these rules; the aim would be to refactor that code at some point.